src/k8s/pkg/k8sd/features/manifests/manifests.go

```diff
@@ -0,0 +1,22 @@
+package manifests
+
+import "github.com/canonical/k8s/pkg/k8sd/types"
+
+// registeredManifests is a list of filesystems that contain charts for features used by k8s-snap.
+var registeredManifests []*types.FeatureManifest
+
+// Manifests returns the list of registered charts.
+func Manifests() []*types.FeatureManifest {
+	if registeredManifests == nil {
+		return nil
+	}
+	manifestList := make([]*types.FeatureManifest, len(registeredManifests))
+	copy(manifestList, registeredManifests)
+	return manifestList
```

Comment:
nit: We can remove the `nil` check if we're okay with returning an empty array instead of a `nil` one. We shouldn't be facing any issue since I think it's recommended to check the `len` of a slice to check if it's empty.
```suggestion
	manifestList := make([]*types.FeatureManifest, len(registeredManifests))
	copy(manifestList, registeredManifests)
	return manifestList
```

Reply:
We can keep this as is, this is more appropriate in terms of returning the zero value (`nil`) of a slice.